src/listeners.js

```diff
@@ -117,6 +175,24 @@ class Listener{
         return !_.isNil(output) && output.toString()
     }
 
+    evalFormList(listValue, func, outputKey) {
+        if (!listValue) return null;
+        var output = []
+        var itemPath = outputKey.slice(outputKey.indexOf(this.listIndexPlaceholder) + this.listIndexPlaceholder.length).split("/")
+        _.each(listValue, function(listItem, index) {
+            var value = listItem[itemPath[0]]
+            var result = null
+            try{
+                result = eval(func)
```

Comment:
Let's leave a note somewhere about looking to replace `eval` before we start trusting input from clients or CS.

Reply:
I added a couple comments to the file